src/views/GovernanceTaskList/SideNavActions/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import './index.scss';
+import Button from 'components/shared/Button';
+
+type SideNavActionsProps = RouteComponentProps;
```

Comment:
Do we expect this component to take all of the props from the RouterComponentProps, or should we pare it down to just history? It looks like that's the only prop we're using?

In code:
```
type SideNavActionProps = {
  history: RouteComponentProps.history;
}
```